.styleci.yml

```diff
@@ -0,0 +1,11 @@
+preset: none
+use-tabs: false
+enabled:
+  - no_trailing_whitespace
+  - no_trailing_whitespace_in_comment
+  - no_whitespace_in_blank_line
+  - single_blank_line_at_eof
+#  - unix_line_endings
```

Comment:
As per config: I added the simplest things, that were done in the original PR. You can find more at [documentation](https://docs.styleci.io/presets), but I'd recommend just enabling the `recommended` or at least `psr12` preset.

`unix_line_endings` is commented because it's highly opinionated and it's up to you to decide.

Reply:
Here is a diff produced for the config I provided: https://github.styleci.io/analyses/QMrVwP